EventProc/src/HcalStripDigiProducer.cxx

```diff
@@ -0,0 +1,141 @@
+#include "TString.h"
+#include "TRandom.h"
+#include "TFile.h"
+#include "TTree.h"
+#include "TClonesArray.h"
+
+#include "Event/EventConstants.h"
+#include "Event/SimCalorimeterHit.h"
+#include "Event/HcalStripHit.h"
+#include "EventProc/HcalStripDigiProducer.h"
+
+#include <iostream>
+
+#include "DetDescr/HcalID.h"
+#include "Event/HcalStripHit.h"
+
+namespace ldmx {
+
+    const float HcalStripDigiProducer::FIRST_LAYER_ZPOS = 569.5;
+    const float HcalStripDigiProducer::LAYER_ZWIDTH = 60.;
+    const int HcalStripDigiProducer::NUM_HCAL_LAYERS = 15;
+    const float HcalStripDigiProducer::MEV_PER_MIP = 1.40;
+    const float HcalStripDigiProducer::PE_PER_MIP = 13.5 * 6. / 4.;
```

Comment:
So looking back at old talks, this should be 13.5 not 13.5 * 1.5...can we fix this and also in HcalDigiProducer.cxx?